test/api/v3/integration/groups/POST-groups.test.js

```diff
@@ -189,6 +189,17 @@ describe('POST /group', () => {
       expect(updatedUser.party._id).to.eql(party._id);
     });
 
+    it('does not award Party Up achievement to solo partier', async () => {
+      await user.post('/groups', {
+        name: partyName,
+        type: partyType,
+      });
+
+      let updatedUser = await user.get('/user');
```

Comment:
instead of updatedUser you can do `await user.sync()` and the original user object will get updated